docs/tutorial/part-two/index.md

```diff
@@ -59,9 +59,9 @@ cd styles
 touch global.css
 ```
 
-> Note: Feel free to create these directories and files using your code editor, if you'd prefer.
+> नोट: अपने कोड एडिटर का उपयोग करके इन फ़ोल्डर और फ़ाइलों को बनाने के लिए आप स्वतंत्र है।
```

Comment:
`नोट: अपने कोड` => `नोट: अगर आप चाहें तो अपने कोड`

how about this? Any thoughts?

Reply:
how about this?
```आप चाहें तो आप अपने मनपसंद कोड एडिटर का उपयोग करके इन फ़ोल्डर और फ़ाइलों को बना सकते है```

Reply:
> how about this?
> `आप चाहें तो आप अपने मनपसंद कोड एडिटर का उपयोग करके इन फ़ोल्डर और फ़ाइलों को बना सकते है`

this one sounds better